src/pages/index.tsx

```diff
@@ -82,6 +85,7 @@ const Home = () => {
   }
   return (
     <CommonPage page={"Home"} backBar={false}>
+      <TopBar />
```

Comment:
Rename component as TutorialTopBar otherwise it could be confused with another topbar